src/comment.rs

```diff
@@ -71,6 +71,16 @@ pub fn rewrite_comment(orig: &str,
     let indent_str = offset.to_string(config);
     let line_breaks = s.chars().filter(|&c| c == '\n').count();
 
+    let num_bare_lines = s.lines()
+        .enumerate()
+        .map(|(_, line)| line.trim())
+        .filter(|l| !(l.starts_with('*') || l.starts_with("//") || l.starts_with("/*")))
+        .count();
+
+    if num_bare_lines > 0 && !config.normalize_comments {
```

Comment:
I'd like to avoid adding more options here in either way, it seems to me that the behaviour of the comment formatting is buggy, rather than it being a behaviour that some people might want. This seems like an improvement, but long term I would like to be able to move a whole comment to indent it more or less, without changing the inside of the comment at all.